types/additional.d.ts

```diff
@@ -1 +1,22 @@
+/**********************************************************************
+ * Copyright (C) 2024-2025 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2025 Red Hat, Inc.
```

Reply:
The file was created in 2024 and was just missing the copyright - thought it's more accurate to show that?